README.md

```diff
@@ -1,15 +1,74 @@
-# Knowledge Base Service
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+<a name="readme-top"></a>
 
-## Execute using Docker Compose
 
-```
-docker compose up -d --build
-```
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/DewyKB/dewy">
+    <img src="images/logo.png" alt="Logo" width="160" height="160">
+  </a>
+
+<h3 align="center">Dewy - The Knowledgebase for AI</h3>
+
+  <p align="center">
+    Opinionated knowledge extraction and semantic retrieval for Gen AI applications.
+    <br />
+    <a href="https://github.com/DewyKB/dewy"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/DewyKB/dewy/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/github_username/repo_name/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Dewy helps you build AI agents and RAG applications by managing the extraction of knowledge from your documents and implementing semantic search over the extracted content. Load your documents and Dewy takes care of parsing, chunking, summarizing, and indexing for retrieval. Dewy builds on the lessons of putting real Gen AI applications into production so you can focus on getting 💩 done, rather than comparing vector databases and building data extraction infrastructure.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Installation
+
+1. Clone the repo
+    ```sh
+    git clone https://github.com/DewyKB/dewy.git
+    ```
+1. Install Python packages
+    ```sh
+    poetry install
+    ```
+1. Configure remote LLM execution (optional - will use local models if not specified)
+    ```js
+    export ENVIRONMENT=LOCAL
+    export REDIS=redis://default:testing123@localhost:6379
```

Comment:
REDIS won't be necessary. DB=`postgresql://...`  will be. I believe we could suggest these go in a `.env` file copied from `env.template` (pydantic-settings can pull them from a file, and falls back to environment)